README.md

```diff
@@ -21,8 +15,23 @@
 
 
 
-<a href="https://www.w3schools.com/html/"><img src="https://img.icons8.com/color/48/000000/html-5--v1.png"/><a>    <a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a>    <a href="https://www.w3schools.com/js/"><img src="https://img.icons8.com/color/48/000000/javascript--v1.png"/><a>    <a href="https://www.w3schools.com/bootstrap/"><img src="https://img.icons8.com/color/48/000000/bootstrap.png"/><a>
-<a href="https://www.tutorialspoint.com/yaml/index.htm"><img class="yaml-logo" width="42px" height="34px" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/5a/Official_YAML_Logo.svg/64px-Official_YAML_Logo.svg.png" alt="Official YAML Logo.svg"/></a>
+<p align="left">
```

Comment:
Can you remove this because it is not needed cuz its already left aligned @AniqJaved